subprojects/shipkit/src/main/groovy/org/shipkit/gradle/configuration/ShipkitConfiguration.java

```diff
@@ -255,6 +255,24 @@ public void setLabelMapping(Map<String, String> labelMapping) {
         public void setIgnoreCommitsContaining(Collection<String> commitMessageParts) {
             store.put("releaseNotes.ignoreCommitsContaining", commitMessageParts);
         }
+
+        /**
+         * Set the Publication Repository where your project can be found
```

Comment:
The publication repository is the place where to look for binaries. It could be MavenCentral, Bintray or some other repository. Can you update the javadoc accordingly? Can you add an example in the Javadoc? See an example if Shipkit project itself.